samples/bot-adaptive-card-actions/csharp/AdaptiveCardActions.sln

```diff
@@ -0,0 +1,25 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
```

Comment:
I can not see the build pipeline updates for this sample.

Reply:
Also, no linking from mail root readme link changes.

Reply:
Deployment pipeline is also not exist.

Reply:
Deployment pipeline, Build Pipeline and Root Readme linked

https://github.com/OfficeDev/Microsoft-Teams-Samples/pull/643